app/scripts/controllers/detect-tokens.test.js

```diff
@@ -18,7 +22,9 @@ describe('DetectTokensController', function () {
   const networkControllerProviderConfig = {
     getAccounts: noop,
   };
-
+  before(function () {
+    globalThis.AbortController = window.AbortController;
+  });
```

Comment:
@NiranjanaBinoy should this be removed?

Reply:
Done 😊 I ended up reverting the changes after resolving it here for some reason I cannot recollect now and missed removing it afterward.😅